### PR TITLE
databricks sample notebook

### PR DESCRIPTION
**Pull request by savitamittal1**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

File: python-sdk/tutorials/automl-with-databricks/automl-databricks-local-with-deployment.ipynb [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "![Impressions](https://PixelServer20190423114238.azurewebsites.net/api/impressions/MachineLearningNotebooks/how-to-use-azureml/azure-databricks/automl/automl-databricks-local-with-deployment.png)"
+      ]
+    }
+  ],
+  "metadata": {
+    "authors": [
+      {
```
Comment (harneetvirk) [inline python-sdk/tutorials/automl-with-databricks/automl-databricks-local-with-deployment.ipynb:551]: Do we still need this metadata?

Comment (savitamittal1) [reply python-sdk/tutorials/automl-with-databricks/automl-databricks-local-with-deployment.ipynb:551]: not sure. It was there in previous github location as well.

Comment (savitamittal1) [reply python-sdk/tutorials/automl-with-databricks/automl-databricks-local-with-deployment.ipynb:551]: I checked other automl notebooks and author is required.

**Review by jeff-shepherd — APPROVED** [review]
How should these be tested?

**Comment by harneetvirk** [comment]
#sign-off

**Review by harneetvirk — APPROVED** [review]

**Comment by savitamittal1** [comment]
> How should these be tested?

We will have to figure out on the testing for databricks notebook as it uses different test framework than automl notebooks.

**Pull request merged**